admin/system/connection.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+    $connection = mysqli_connect("localhost", "root", "", "pindad-db");
+    if(mysqli_connect_errno()){
+        echo mysqli_connect_errno();
```

Comment:
Codacy found an issue: [The use of function mysqli_connect_errno() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)